src/target/adiv5_swd.c

```diff
@@ -427,6 +428,7 @@ uint32_t firmware_swdp_low_access(adiv5_debug_port_s *dp, const uint8_t RnW, con
 			DEBUG_ERROR("SWD access resulted in parity error\n");
 			raise_exception(EXCEPTION_ERROR, "SWD parity error");
 		}
+		swd_proc.seq_out(0, 8U);
```

Comment:
Rather than inserting this here, let's move the comment and call from the else below to just above `return response` (and eliminate the else's braces)? Saves a duplicated call, and puts the explainer back with it